deepchem/data/datasets.py

```diff
@@ -915,21 +949,35 @@ def to_json(self, fname: str) -> None:
 
   @staticmethod
   def from_json(fname: str) -> "NumpyDataset":
+    """Create NumpyDataset from the json file.
+
+    Parameters
+    ----------
+    fname: str
+      The name of the json file.
+
+    Returns
+    -------
+    NumpyDataset
+      A new NumpyDataset created from the json file.
+    """
     with open(fname) as fin:
       d = json.load(fin)
       return NumpyDataset(d['X'], d['y'], d['w'], d['ids'])
 
   @staticmethod
   def merge(datasets: Sequence[Dataset]) -> "NumpyDataset":
-    """
+    """Merge multiple NumpyDatasets.
+
     Parameters
     ----------
-    datasets: list of deepchem.data.Dataset
-      list of datasets to merge
+    datasets: List[Dataset]
+      List of datasets to merge.
 
     Returns
     -------
-    Single deepchem.data.NumpyDataset with data concatenated over axis 0
+    NumpyDataset
+      A single deepchem.data.NumpyDataset with data concatenated over axis 0.
```

Comment:
A clearer description would be, "A single deepchem.data.NumpyDataset containing all the samples from all datasets."